app_dart/lib/src/service/config.dart

```diff
@@ -306,6 +306,9 @@ class Config {
   static RepositorySlug get packagesSlug => RepositorySlug('flutter', 'packages');
   static RepositorySlug get pluginsSlug => RepositorySlug('flutter', 'plugins');
 
+  /// Flutter recipes is hosted on Gerrit instead of GitHub.
+  static RepositorySlug get recipesSlug => RepositorySlug('flutter', 'recipes');
```

Comment:
I think if we add more GoB functionality to Cocoon, we should create an extension to this so the type is explicit about GitHub vs GoB